poetry.lock

```diff
@@ -1907,52 +1908,52 @@ files = [
 
 [[package]]
 name = "matplotlib"
-version = "3.7.2"
+version = "3.7.1"
```

Comment:
This PR would downgrade matplotlib. There is currently a dependency conflict between the latest version of pyparsing and the latest version of matplotlib.